docs/operate/mobility/move-base.md

```diff
@@ -5,5 +5,5 @@ weight: 40
 layout: "docs"
 type: "docs"
 no_list: true
-description: "TODO"
+description: "Move a mobile robot with manual or autonomous navigation."
```

Comment:
I think robot is good in this context because "machines" that move around are often thought of as robots, and this is the non-Viam-specific way to describe the task. There are exceptions, but I think "robot" will create the best understanding in this case.